eng/pipelines/templates/jobs/archetype-go-release.yml

```diff
@@ -24,10 +24,14 @@ stages:
                 steps:
                   - checkout: self
                   - template: /eng/common/pipelines/templates/steps/retain-run.yml
+                  - template: /eng/common/pipelines/templates/steps/verify-changelog.yml
+                    parameters:
+                      ServiceName: ''
```

Comment:
@chidozieononiwu this is a workaround until https://github.com/Azure/azure-sdk-tools/pull/2016 can get merged.